Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.h

```diff
@@ -323,6 +337,20 @@ class Arm64FPRCache : public Arm64RegCache
 
   void StoreRegisters(BitSet32 regs) { FlushRegisters(regs, false); }
 
+  Arm64Gen::ARM64Reg GetConstant(
+      u64 value, bool single, bool want_duplicated, bool generates_duplicated,
+      const std::function<void(Arm64Gen::ARM64Reg, Arm64Gen::ARM64FloatEmitter*)>& generate);
+
+  // Equivalent to calling GetConstant two times, except if both constants need to be generated,
+  // generate_both is used instead of generate1 and generate2.
+  std::pair<Arm64Gen::ARM64Reg, Arm64Gen::ARM64Reg> GetTwoConstants(
```

Comment:
Perhaps the function params should be assigned aliases to make this a little nicer to read. Currently this is quite noisy to visually scan through at once.